Documentation/ceph-object-store-user-crd.md

```diff
@@ -33,3 +39,7 @@ spec:
 
 * `store`: The object store in which the user will be created. This matches the name of the objectstore CRD.
 * `displayName`: The display name which will be passed to the `radosgw-admin user create` command.
+* `maxBuckets`: The maximum number of buckets the user can create.
+* `maxSize`: The maximum size of buckets the user can consume.
+* `maxObjects`: The maximum number of objects the user can create.
```

Comment:
Should we document that `-1` means no maximum? You use this later in objectstore_test.go, but it's not documented.

Reply:
will provide link for RGW documentation